kubernetes/host-logs-geneva.yaml

```diff
@@ -56,10 +56,13 @@ spec:
        livenessProbe:
           exec:
             command:
-              - cmd
-              - /c
-              - .\opt\hostlogswindows\scripts\cmd\livenessprobe.exe
-              - ".\\etc\\hostlogswindows\\filesystemwatcher.txt"
+              - C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe
+              - -command
+              - |
+                .\opt\hostlogswindows\scripts\powershell\livenessprobe.ps1 -monAgentManagerExePath "\opt\genevamonitoringagent\genevamonitoringagent\Monitoring\Agent\MonAgentManager.exe" -fileSystemWatcherTextFilePath "etc\hostlogswindows\filesystemwatcher.txt"
+          initialDelaySeconds: 60
+          periodSeconds: 180
```

Comment:
Per offline conversation these two values should be swapped so the probe period isn't 3 minutes (resulting in a total of 9 minutes before failure would be detected): https://github.com/microsoft/Docker-Provider/blob/ca2b6783558a9b86d4dd28b4bfb895d95a3776ad/kubernetes/ama-logs.yaml#L1007